packages/php/email-editor/tests/integration/Integrations/Core/Renderer/Blocks/Embed_Test.php

```
@@ -579,4 +533,168 @@ public function test_youtube_embed_falls_back_to_link_when_thumbnail_fails(): vo
+		// Return a thumbnail URL with query parameters to test the URL encoding fix.
+		$mock_thumbnail_url   = 'https://videos.files.wordpress.com/BZHMfMfN/thumbnail.jpg?w=500&h=281';
+		$mock_oembed_response = wp_json_encode(
+			array(
+				'type'          => 'video',
+				'thumbnail_url' => $mock_thumbnail_url,
+				'title'         => 'Test Video',
+			)
+		);
+
+		// Use pre_http_request filter to intercept oEmbed HTTP calls.
+		$filter_callback = function ( $preempt, $args, $url ) use ( $mock_oembed_response ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$args'. (undefined)

(UnusedFormalParameter)